src/main/kotlin/com/github/suusan2go/kotlinfillclass/inspections/FillClassInspection.kt

```diff
@@ -108,7 +111,7 @@ class FillClassFix(
             if (argumentExpression is KtQualifiedExpression || argumentExpression is KtLambdaExpression) {
                 ShortenReferences.DEFAULT.process(argumentExpression)
             }
-            if (argumentExpression != null && withTrailingComma && index == parameters.lastIndex) {
+            if (argumentExpression != null && index == parameters.lastIndex && withTrailingComma && !hasTrailingComma) {
```

Comment:
Since `if` condition has been longer, I thought it would become nicer if define a variable like 
```kotlin
val needsTrailingComma = withTrailingComma && !hasTrailingComma()
```
 instead of `hasTrailingComma` variable.